uint32 	msg_counter
uint8 	drivingMode